.storybook/src/accessibility-tool.tsx

```diff
@@ -0,0 +1,38 @@
+import React, {useCallback} from 'react'
+import {useGlobals} from '@storybook/api'
+import {ThemeProvider, ToggleSwitch, Box} from '../../src'
+
+export const ADDON_ID = 'show-surrounding-links'
+export const TOOL_ID = `${ADDON_ID}/tool`
+
+export const Tool = () => {
+  const [{showSurroundingElements, colorScheme}, updateGlobals] = useGlobals()
+
+  const toggleMyTool = useCallback(() => {
+    const currentValue = showSurroundingElements ?? window?.localStorage.getItem('showSurroundingElements') === 'true'
+
+    window?.localStorage?.setItem('showSurroundingElements', `${!currentValue}`)
+
+    updateGlobals({
+      showSurroundingElements: !currentValue
+    })
+  }, [showSurroundingElements])
+
+  return (
+    <ThemeProvider colorMode="day" dayScheme={colorScheme}>
+      <Box style={{display: 'flex', alignItems: 'center'}}>
+        <Box fontSize={0} fontWeight="bold" id={`${ADDON_ID}-label`} color="black">
+          Wrap links
```

Comment:
I'll leave a separate comment with another design consideration here, but if we do keep ToggleSwitch I think the label "Links before/after" might be better (I workshopped this with a few designers.)

Worth noting "Link sandwich" came up 🥪

Reply:
OH MY GOSH CUTE! I wish we had a cute sandwich icon to add.

Reply:
In all seriousness, I could whip up a lil' SVG sandwich if you wanted 😅

Reply:
I'm just gonna leave this here:

```
<svg aria-hidden="true" focusable="false" xmlns="http://www.w3.org/2000/svg" width="101" height="64" fill="none"><path fill="currentColor" d="M95.547 19.863v-3.205a4.026 4.026 0 0 0-2.66-3.788L59.222.73a4.025 4.025 0 0 0-2.731 0L24.209 12.37a37.833 37.833 0 0 0-2.662-.091c-3.82 0-7.444.559-10.2 1.573-5.777 2.126-6.64 5.774-6.64 7.759v4.837c-3.149 2.13-3.714 4.79-3.714 6.393 0 1.447.503 4.228 3.816 6.438.126.776.389 1.5.764 2.18v3.574c0 5.76 6.454 9.337 16.84 9.337.662 0 1.32-.018 1.973-.051 1.317 6.71 10.978 9.117 19.71 9.117 8.564 0 18.02-2.315 19.622-8.736l30.035-10.834a4.024 4.024 0 0 0 2.66-3.787v-5.871c2.945-1.658 4.58-4.032 4.58-6.955.001-3.19-1.961-5.724-5.446-7.39Zm-86.722 1.05c.02-.073.052-.142.077-.213.051-.15.108-.296.183-.437.042-.08.094-.155.144-.231.042-.063.091-.124.137-.187.045-.062.086-.126.137-.186.062-.075.129-.148.2-.22.055-.06.122-.115.183-.171.062-.056.12-.115.186-.17.078-.065.152-.133.237-.194a6.37 6.37 0 0 1 .497-.335c.078-.05.151-.101.233-.148.133-.077.281-.145.42-.216.14-.073.271-.148.42-.216l.03-.013.01-.004c.289-.129.593-.251.909-.365.107-.038.222-.073.332-.11h.004c.229-.08.46-.153.703-.222.137-.039.276-.075.415-.111h.002c.23-.06.465-.119.705-.172h.001c.148-.034.297-.065.448-.097h.001c.084-.017.168-.031.253-.047.166-.032.334-.064.503-.094.15-.025.296-.052.449-.076.083-.012.166-.021.248-.032.205-.03.408-.06.617-.084.127-.016.252-.035.379-.049.422-.045.849-.086 1.28-.116.03-.002.06-.002.09-.006h.002c.4-.026.804-.046 1.208-.058.144-.004.287-.005.431-.007.28-.005.56-.008.837-.007.342.002.683.012 1.024.024.164.005.33.007.492.016.503.025 1.002.059 1.494.105L57.852 4.526l33.664 12.14-3.447 1.245-9.278 3.346-3.771 1.36-.434.156-1.73.624-3.772 1.36-.434.156-4.645 1.675-.009.003-5.107 1.842c.097.322.163.655.163 1.006 0 .26-.031.51-.086.756-.018.08-.05.156-.073.233-.05.162-.1.321-.171.475-.039.089-.093.173-.142.26a3.898 3.898 0 0 1-.249.405c-.063.086-.132.171-.203.254a4.787 4.787 0 0 1-.583.604 6.894 6.894 0 0 1-.407.336c-.099.074-.194.147-.297.219a10.16 10.16 0 0 1-.494.314c-.107.063-.212.131-.324.192-.19.104-.396.204-.6.303-.107.053-.21.106-.32.156-.258.115-.53.226-.809.33-.07.027-.136.06-.208.084-.35.128-.713.248-1.088.361-.134.04-.276.075-.412.113-.25.07-.497.14-.754.203-.182.045-.369.084-.553.126-.222.049-.443.1-.67.146-.207.04-.42.078-.633.115-.21.038-.42.073-.636.106-.233.036-.466.07-.702.103-.198.026-.397.051-.595.075-.255.03-.513.06-.772.084-.178.018-.356.032-.537.05-.284.021-.567.045-.855.064-.141.008-.283.015-.424.02-.327.02-.653.034-.982.044-.045.002-.089.002-.135.002-3.924.102-7.94-.338-11.073-1.318l-.011-.003c-.325-.103-.64-.21-.945-.325-.064-.022-.126-.049-.19-.074a13.877 13.877 0 0 1-1.53-.693c-.086-.047-.173-.093-.256-.14-.173-.1-.337-.202-.498-.305-.08-.054-.16-.104-.236-.155a7.334 7.334 0 0 1-.452-.348c-.06-.049-.126-.096-.183-.146a5.776 5.776 0 0 1-.536-.526c-.07-.08-.15-.154-.212-.236a3.745 3.745 0 0 1-.731-1.55 3.583 3.583 0 0 1-.07-.57c-.002-.05-.017-.097-.017-.148 0-.298.046-.583.118-.86v-.002c.059-.23.135-.456.24-.671.064-.134.14-.264.223-.392.396-.625.98-1.186 1.723-1.684-.556.176-1.15.33-1.768.463l-.154.034c-.62.13-1.261.241-1.925.33-.004 0-.008 0-.012.002-.662.087-1.341.151-2.03.197l-.174.011c-.69.042-1.389.065-2.087.065l-.048-.001a32.866 32.866 0 0 1-2.154-.073c-.092-.006-.185-.011-.277-.018a30.059 30.059 0 0 1-1.038-.092c-.069-.007-.138-.017-.208-.024a27.1 27.1 0 0 1-.824-.104l-.31-.045-.266-.044c-.183-.03-.365-.06-.544-.092l-.188-.033a23.05 23.05 0 0 1-1.182-.264 22.09 22.09 0 0 1-.683-.184c-.082-.03-.167-.053-.251-.078-.287-.088-.568-.18-.836-.278l-.022-.01c-.26-.096-.507-.199-.749-.305a6.662 6.662 0 0 1-.21-.099 9.469 9.469 0 0 1-.537-.27c-.057-.033-.118-.063-.172-.095a7.668 7.668 0 0 1-.609-.387c-.036-.025-.067-.054-.103-.08-.074-.055-.14-.114-.209-.172-.073-.06-.149-.117-.217-.178a4.417 4.417 0 0 1-.487-.515c-.023-.029-.051-.058-.073-.087a3.252 3.252 0 0 1-.307-.497 2.112 2.112 0 0 1-.06-.136 2.873 2.873 0 0 1-.147-.416c-.014-.053-.03-.105-.041-.16a2.737 2.737 0 0 1 .025-1.278Zm88.057 7.04c-.02.077-.054.148-.078.22-.048.146-.094.291-.163.43-.042.084-.102.164-.151.246a3.494 3.494 0 0 1-.236.358c-.007.01-.016.017-.023.026-.06.075-.128.145-.194.216-.097.106-.193.213-.306.315-.086.08-.186.154-.279.23-.1.08-.2.159-.307.234-.021.015-.039.031-.062.046a6.661 6.661 0 0 1-.338.214c-.048.03-.096.056-.145.085-.093.055-.18.112-.28.165-.124.067-.256.133-.39.197-.153.076-.311.15-.475.222a19.76 19.76 0 0 1-.957.371 18.007 18.007 0 0 1-1.55.461c-.195.05-.398.097-.603.143a18.4 18.4 0 0 1-.488.106l-.044.009c-.176.035-.353.065-.527.097l-.11.021a19.586 19.586 0 0 1-1.227.186c-.186.024-.371.05-.561.07-.233.027-.47.049-.707.07-.188.017-.371.035-.562.05-.057.003-.113.005-.17.01l-.013.003c-.187.013-.376.021-.566.03-.18.01-.358.021-.54.027-.27.01-.54.014-.811.018-.163.002-.328.008-.493.008h-.045c-.415 0-.83-.007-1.244-.02-.234.656-.71 1.243-1.361 1.762-1.127.893-2.793 1.573-4.752 2.043-.187.896-.446 1.7-.777 2.356-.521 1.037-1.222 1.695-2.103 1.695-.456 0-.86-.181-1.221-.494-.493-.43-.895-1.121-1.207-1.973-.04-.106-.081-.211-.118-.324a3.816 3.816 0 0 1-.37.5c-1.122 1.313-3.294 2.258-5.936 2.834-.702 1.343-2.42 2.36-4.662 3.059-.261.08-.522.159-.796.232-.046.743-.37 1.405-.903 1.996l-.002.003c-1.714 1.888-5.639 2.991-9.943 3.306-.46 1.06-1.016 1.933-1.665 2.54-.65.607-1.392.95-2.23.95-.783 0-1.478-.313-2.097-.846-.755-.652-1.391-1.646-1.901-2.887l-.03-.065c-2.648-.423-4.971-1.18-6.459-2.271-1.2-.142-2.35-.35-3.413-.618-1.062-.272-2.03-.607-2.867-1.007-1.308-.623-2.29-1.402-2.776-2.34a2.93 2.93 0 0 1-.162-.354c-4.548-.272-8.755-1.41-10.486-3.416l-.004-.008c-.385-.446-.65-.938-.766-1.47a23.658 23.658 0 0 1-2.796-.613l-.015-.003a17.906 17.906 0 0 1-.657-.203 15.301 15.301 0 0 1-.754-.269l-.014-.006c-1.158-.45-2.124-1.003-2.814-1.657a4.102 4.102 0 0 1-.548-.616l-.006-.01c-.389-.545-.606-1.146-.606-1.808 0-.229.026-.452.076-.667.018-.07.05-.138.073-.208.043-.141.084-.282.148-.416.01-.02.025-.039.036-.058.202.431.462.84.755 1.232.155.21.313.416.493.611.147.162.31.314.47.468.704.66 1.554 1.246 2.545 1.755l.006.004c.099.051.2.1.301.149l.011.006c.19.09.375.187.573.27a.016.016 0 0 0 .006.004c2.815 1.218 6.557 1.888 11.037 1.888.66 0 1.32-.02 1.973-.051.202 1.033.622 1.95 1.186 2.787a9.564 9.564 0 0 0 2.26 2.334l.02.014c.372.273.766.529 1.18.772l.171.09c3.912 2.215 9.568 3.117 14.89 3.117 6.184 0 12.816-1.215 16.638-4.301.313-.254.607-.52.88-.798 1.012-1.031 1.757-2.229 2.106-3.635l21.34-7.697.435-.156 5.501-1.984.436-.157 2.322-.838a4.018 4.018 0 0 0 2.543-2.873c.06.048.118.095.174.143.084.071.17.142.247.217.104.1.19.205.28.31.064.076.137.15.192.229.085.113.148.23.217.349.043.076.097.15.134.229.062.132.105.272.146.412.022.07.056.137.07.206.05.213.075.434.075.662.001.243-.029.474-.084.7Z"/></svg>
```